src/openloco/station.cpp

```diff
@@ -237,7 +237,7 @@ namespace openloco
             }
         }
 
-        if (var_2A != 384 && is_player_company(owner))
+        if (flags != (station_flags::flag_7 | station_flags::flag_8) && is_player_company(owner))
```

Comment:
This implementation is wrong. It should check if flags 7 and 8 are both not set. 
![image](https://user-images.githubusercontent.com/737603/57810210-88aeba80-7767-11e9-8237-0af0214e3741.png)

Reply:
Ah, nice catch! So it should be the following?
```suggestion
        if ((flags & (station_flags::flag_7 | station_flags::flag_8)) != 0 && is_player_company(owner))
```